test/integration/deploy_int_test.go

```diff
@@ -313,6 +333,10 @@ func (suite *DeployIntegrationTestSuite) AssertConfig(ts *e2e.Session, targetID
 		rcFile, err := subshell.RcFile()
 		suite.Require().NoError(err)
 
+		if fileutils.FileExists(filepath.Join(ts.Dirs.HomeDir, filepath.Base(rcFile))) {
```

Comment:
Note there is no mention of `user.HomeDir()` like in the previous instance. This is what I'd prefer to see.

Reply:
The other instance is still using `user.HomeDir()` under the hood, at least for the bash implementation of the `subshell`.